resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/refresh/RefreshManager.java

```diff
@@ -12,9 +12,11 @@
  *     IBM - Initial API and implementation
  *     Christoph Läubrich 	- Issue #84 - RefreshManager access ResourcesPlugin.getWorkspace in the init phase
  *     						- Issue #97 - RefreshManager.manageAutoRefresh calls ResourcesPlugin.getWorkspace before the Workspace is fully open
+ *     Latha Patil(ETAS GmbH)	- GitHub Issue #497- Get rid of deprecated org.eclipse.core.runtime.Preferences in platform code
```

Comment:
Adding the contribution information is optional. If you want to keep it, please also change the year in the first line from 2022 to 2023. 
Nitpicky: I would prefer to format the copyright line consistent with the existing ones, i.e., only containing "Issue ..." instead of "GitHub Issue ...".

Reply:
Thanks for the review . I will adopt the changes and provide for re-review